tests/vmi_configuration_test.go

```diff
@@ -2110,4 +2110,19 @@ var _ = Describe("Configurations", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 	})
+
+	FIt("Masquerede VMI with Guest agent should not update VMI IP", func() {
+		vmi := tests.NewRandomFedoraVMIWitGuestAgent()
+
+		By("Starting a VirtualMachineInstance")
+		vmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+		Expect(err).ToNot(HaveOccurred())
+		tests.WaitForSuccessfulVMIStart(vmi)
+
```

Comment:
@vatsalparekh Maybe its better to do another check using
```
tests.WaitAgentConnected(virtClient, vmi)
```
 thus making sure that GA is up.